bin/pca-tool

```diff
@@ -419,6 +420,21 @@ def delete_image(image_id):
     return True
 
 
+def output_user_report(customer_id, assessment_id):
```

Comment:
Since this function can output more than one user report, I'd call it `output_user_reports`:
```suggestion
def output_user_reports(customer_id, assessment_id):
```

Don't forget to update any other references to this function accordingly.